R/assocTestAggregate.R

```diff
@@ -148,15 +148,15 @@ setMethod("assocTestAggregate",
           function(gdsobj, null.model, AF.max=1,
                    weight.beta=c(1,1), weight.user=NULL,
                    test=c("Burden", "SKAT", "fastSKAT", "SMMAT", "SKATO"),
-                   burden.test=c("Score", "Wald"),
+                   # burden.test=c("Score"),
```

Comment:
@mconomos what do you think about removing this argument since it only has one option now?

Reply:
I was originally thinking we would leave it because BinomiRare and CMP would be options once we incorporate them. However, looking at that code again, I think we could just make them `test` options. Whatever you think would be better.

Reply:
The more I think about it, the more I lean toward making those `test` options.